### ISSUE TITLE
choose linux distro for ecology and economy?

### ISSUE THREAD

**Issue by ghost** [distrochooser/distrochooser#232]
**Is your feature request related to a problem? Please describe.**
None.

**Describe the solution you'd like**
I would like to know which Linux Distro spends less ram memory processing or at the same time spends less energy. I think this information is important to prevent me from spending money or some natural resource or avoiding waste.

**Describe alternatives you've considered**
Generally things with the Raspberry-pi with Debian or Ubuntu are great for automating things around the house and don't use a lot of power or ram. Examples:
- Snakeware-os, Raspeberry-pi(with Debian, Ubuntu etc), tinyware, Absolute linux or
- Tiny core linux, Porteus, Puppy linux, Slitaz, AntiX Linux, Bodhi Linux, Linux Lite, ArchBang

*In other words, choose a linux distro for embedded systems or linux distro with less memory*

**Additional context**
None.

**Comment by cmllr** [comment]
hey @eantho 

thanks for the suggestion! I think this is an important question to ask for. An issue I see here: The Distributions you mentioned here all require some experience or work in a niche (TCL, Puppy). I removed Puppy a few years back because of this.

An _idea_: Add a flag to #220 like "Prefer distribution with low requirements" to allow the user to filter distribution by own hardware requirements and/ or to add a sorting layer based on requirements (currently, #220 only defines a filter, not a sorting).

As a second step, I would see some additions to the distro list, for example on Bodhi Linux.

For Raspberry OS'es I think it would be required to have a flag checking form arm(v7) architecture first.

What do you think?

**Comment by cmllr** [comment]
Closing this as reporter is not existing anymore.
